src/types/onyx/Request.ts

```diff
@@ -11,7 +11,7 @@ type RequestData = {
     command: string;
     commandName?: string;
     data?: Record<string, unknown>;
-    type?: string;
+    type?: 'get' | 'post';
```

Comment:
I see that this type (`"get" | "type"`) is also used in `xhr` for example. Can we create one type to re-use it?

Reply:
Should I wait for this change before testing? cc @fvlvte